### ISSUE TITLE
In 11.2.4 update - Error thrown calling textlint: 'normalizer is not a function'

### ISSUE THREAD

**Issue by jennifer-shehane** [textlint/textlint#599]
**What version of textlint are you using?**

11.2.4

**What file type (Markdown, plain text, etc.) are you using?**

Markdown

**What did you do? Please include the actual source code causing the issue.**

Updated from 11.2.3 to 11.2.4.

Called `textlint source/*` - `source/*` directory contains all Markdown files - did not throw error in 11.2.3

Source code calling textlint: https://github.com/cypress-io/cypress-documentation/blob/develop/package.json#L31

**Failed CI Error:**

```
✖ Error
normalizer is not a function

✖ Stack trace
TypeError: normalizer is not a function
```

![Screen Shot 2019-04-30 at 11 37 16 AM](https://user-images.githubusercontent.com/1271364/56941819-ed013580-6b3c-11e9-9f28-edc72d810541.png)

**Failed CI Error Link:** https://circleci.com/gh/cypress-io/cypress-documentation/12575

**What did you expect to happen?**

Do not throw error on `textlint`

**What actually happened? Please include the actual, raw output from textlint.**

**Debug logs:**

```
 npm run md-lint

> cypress-documentation@0.0.0 md-lint /Users/jennifer/Dev/cypress-documentation
> textlint --debug source/*

  textlint:cli textlint --version: 11.2.4 +0ms
  textlint:cli Running on files, stdin-filename: undefined +1ms
✖ Error
normalizer is not a function

✖ Stack trace
TypeError: normalizer is not a function
    at /Users/jennifer/Dev/cypress-documentation/node_modules/textlint/lib/textlint/src/config/config.js:21:16
    at Array.map (<anonymous>)
    at applyNormalizerToList (/Users/jennifer/Dev/cypress-documentation/node_modules/textlint/lib/textlint/src/config/config.js:20:18)
    at new Config (/Users/jennifer/Dev/cypress-documentation/node_modules/textlint/lib/textlint/src/config/config.js:113:22)
    at Function.Config.initWithAutoLoading (/Users/jennifer/Dev/cypress-documentation/node_modules/textlint/lib/textlint/src/config/config.js:289:16)
    at Function.Config.initWithCLIOptions (/Users/jennifer/Dev/cypress-documentation/node_modules/textlint/lib/textlint/src/config/config.js:212:21)
    at Object.executeWithOptions (/Users/jennifer/Dev/cypress-documentation/node_modules/textlint/lib/textlint/src/cli.js:107:38)
    at Object.execute (/Users/jennifer/Dev/cypress-documentation/node_modules/textlint/lib/textlint/src/cli.js:94:25)
    at /Users/jennifer/Dev/cypress-documentation/node_modules/textlint/bin/textlint.js:37:20
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:611:11)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:598:3
```

**Comment by azu** [comment]
Thanks for report!

Probably, following commit is the cause of this issue.

https://github.com/textlint/textlint/commit/993080965c679f340d3a71c73c72f4cbe29bd98e.

`textlint@11.2.4` depended on `@textlint/type@1.1.4` internally.
But, your `package-lock.json` lock `@textlint/type@1.1.3`.
It will cause *normalizer is not a function* error.

https://github.com/cypress-io/cypress-documentation/blob/1714a6e826cfa39f07f8636e5b7345f30da65064/package-lock.json#L2431-L2440

I've noticed that `textlint` does not defined `@textlint/types` as `dependencies`.
It is a bug 🐛 

https://github.com/textlint/textlint/blob/bda41c2f7dadc3f69980385a88ad27ccac7a8e92/packages/textlint/package.json#L50-L81

We will fix it soon.

**Comment by azu** [comment]
@jennifer-shehane I've released https://github.com/textlint/textlint/releases/tag/textlint%4011.2.5

Please try it :)

```
npm i -D textlint@latest
```

**Comment by jennifer-shehane** [comment]
Thanks for the quick response. The update works. 👍
